rst2pdf/style-to-yaml.py

```diff
@@ -0,0 +1,25 @@
+#!/usr/bin/env python
```

Comment:
`python3`

On all Linux distros except for Arch, `python` historically points to `python2.7`. This is changing but it's still safer to be explicit IMO

Reply:
Also, license information? Even a SPDX header would do the trick.

```
# SPDX-License-Identifier: MIT
```

Reply:
Yes to the license headers! We must try to unblock your PR for those across the rest of the project too